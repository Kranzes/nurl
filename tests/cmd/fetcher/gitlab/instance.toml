args = ["https://gitlab.gnome.org/GNOME/libadwaita", "1.2.0"]
