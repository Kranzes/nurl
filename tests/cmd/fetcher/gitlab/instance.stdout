fetchFromGitLab {
  domain = "gitlab.gnome.org";
  owner = "GNOME";
  repo = "libadwaita";
  rev = "1.2.0";
  hash = "sha256-3lH7Vi9M8k+GSrCpvruRpLrIpMoOakKbcJlaAc/FK+U=";
}